src2/components/Counter/Counter.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+class Counter extends Component {
+  state = {
+    count: this.props.value,
+  }
+
+  render() {
+    return <div className={this.getCounterClasses()}>{this.formatCount()}</div>
+  }
+
+  getCounterClasses = () => {
```

Comment:
As already written the comment above, you can simplify your code and make it easier to comprehend:
```suggestion
  getCounterClasses = () => 
    `p-2 badge badge-${this.props.value >= 3 ? 'warning' : 'primary'}`
```